hive_agent/agent.py

```diff
@@ -26,6 +27,9 @@
 logging.basicConfig(stream=sys.stdout, level=logging.INFO)
 logging.getLogger().addHandler(logging.StreamHandler(stream=sys.stdout))
 
+dir_path = os.path.dirname(os.path.realpath(__file__))
+config_path = os.path.join(dir_path, '..', 'settings.toml')
```

Comment:
many projects use `settings.toml` so we will have to use a more descriptive name like `hive-config.toml`